Formula/cp2k.rb

```diff
@@ -5,6 +5,14 @@ class Cp2k < Formula
   sha256 "dff343b4a80c3a79363b805429bdb3320d3e1db48e0ff7d20a3dfd1c946a51ce"
   license "GPL-2.0-or-later"
 
+  # Upstream uses GitHub releases to indicate that a version is released
+  # (there's also sometimes a notable gap between when a version is tagged and
+  # and the release is created), so the `GithubLatest` strategy is necessary.
```

Comment:
```suggestion
```

This formula is using a GitHub release asset, so this should be using the `GithubLatest` strategy anyway. I usually reserve the explanatory comment for when we're using one of the GitHub strategies but `stable` is a tag or tag archive, so we can omit it here.